sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -102,18 +102,18 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
   }
 
   test("uncache of nonexistent tables") {
-    val expectedErrorMsg = "Table or view not found: nonexistentTable"
+    val expectedErrorMsg = "Table or view not found:"
     // make sure table doesn't exist
     var e = intercept[AnalysisException](spark.table("nonexistentTable")).getMessage
-    assert(e.contains(expectedErrorMsg))
+    assert(e.contains(s"$expectedErrorMsg nonexistentTable"))
```

Comment:
`spark.table` just creates `UnresolvedRelation(multipartIdentifier)`, so it doesn't add current namespace. 

Perhaps, should we retaine the original multipart names in `UnresolvedRelation`, etc. and use it inside `CheckAnalysis`? For `SELECT * FROM tbl`, what is a more desirable message, `Table or view not found: tbl` or `Table or view not found: default.tbl`?

Note that this PR always adds the current namespace to the identifier and postgres seems to use the original identifier given.